package.json

```diff
@@ -3,22 +3,33 @@
   "description": "Internet website for the ReactJS Dallas User Group",
   "version": "1.0.0",
   "author": "Mike Mathew <m2mathew@me.com>",
+  "engines": {
+    "node": "^8.15.1",
+    "npm": "6.x"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/reactjs-dallas/reactjs-dallas-site"
+  },
   "dependencies": {
+    "@babel/polyfill": "^7.2.5",
     "@material-ui/core": "^3.6.1",
+    "@material-ui/icons": "^3.0.1",
     "dotenv": "^6.1.0",
     "firebase": "^5.5.3",
     "gatsby": "^2.0.64",
     "gatsby-plugin-glamor": "^2.0.7",
-    "gatsby-plugin-manifest": "^2.0.11",
-    "gatsby-plugin-offline": "^2.0.19",
     "gatsby-plugin-react-helmet": "^3.0.4",
+    "gatsby-plugin-remove-serviceworker": "^1.0.0",
```

Comment:
😱

Reply:
just messin' :)

Reply:
It's a limitation of using Material-UI alongside Gatsby. It was a painful line to comment out. 🤕 

I think it can be added back in with someone who actually knows what they are doing with regards to service workers. 😜